crates/operator-wallet/src/lib.rs

```diff
@@ -150,12 +146,14 @@ impl OperatorWallet {
             .into_iter()
             .map(|lo| lo.outpoint)
             .collect();
+
         let mut tx_builder = self.general_wallet.build_tx();
         // DON'T spend any of the cpfp outputs
         tx_builder.unspendable(cpfp_utxos);
         tx_builder.fee_rate(fee_rate);
-        tx_builder.add_recipient(user_p2tr_address.script_pubkey(), amount);
+        tx_builder.add_recipient(user_script_pubkey, amount);
```

Comment:
Just an FYI, `bdk_wallet` 1.2.0 relaxes the constraints around `add_recipient`: https://github.com/bitcoindevkit/bdk/pull/1841